src/discussions/navigation-bar/NavigationBar.jsx

```diff
@@ -0,0 +1,62 @@
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import Button from '@edx/paragon/dist/Button';
+import SearchField from '@edx/paragon/dist/SearchField';
+import React from 'react';
+import { useParams } from 'react-router';
+import { NavLink } from 'react-router-dom';
+import { Routes } from '../../data/constants';
+import messages from '../../data/messages';
+
+function NavigationBar({ intl }) {
+  const { courseId } = useParams();
+  return (
+    <div className="navigation-bar d-flex flex-row my-2">
```

Comment:
After clicking on a thread, the navigation buttons loose the indication of which page the user is in.
![image](https://user-images.githubusercontent.com/27893385/91783726-6c8e9100-ebd7-11ea-8866-94f28f2f52bd.png)

Reply:
These should implemented using the Nav component from Paragon:
https://edx.github.io/paragon/components/navs.

This was already written down in mockups as `Developer Notes`. Is there any issue in using the recommended components?
https://www.figma.com/proto/LVFnO9oRHYiamoicZuXDAC/Discussion-page?node-id=298%3A0&scaling=min-zoom
![image](https://user-images.githubusercontent.com/27893385/91784545-8b8e2280-ebd9-11ea-92c3-a49021613098.png)

Reply:
@giovannicimolin

We are currently using edx/paragon version `10.0.1`, which does not have the `Nav` component. Please see [here|https://github.com/edx/paragon/tree/v10.0.1/src]

Reply:
> We are currently using edx/paragon version `10.0.1`, which does not have the `Nav` component. Please see [here](https://github.com/edx/paragon/tree/v10.0.1/src)

@pcockwell Yes it does: https://github.com/edx/paragon/blob/v10.0.1/src/index.js#L106
If you visit the [link mentioned on the notes page](https://edx.github.io/paragon/components/navs) you will learn that it is a pass-through component to React-Bootstrap.

Reply:
@xitij2000 @giovannicimolin

I've been playing around with the `Nav` component. I don't particularly think the user experience on it is very good. When you click an element in the `Nav`, it actually does a full page reload as it navigates to that URL, where as the `NavLink` code I had previously, uses react routing to send the user to that page. Further, without using SCSS I cannot find a way to style the `active` element, and the current classes have it simply be a black text item (as opposed to the default blue) and does not have any background color.

My recommendation is to remain with the current `NavLink` implementation, though if desired I can push a change that will use the `Nav` component so you can compare the results.

Reply:
That happens because the Nav component uses the `a` tag by default. However, it includes a mechanism to provide your own link component via the `as` attribute. For instance you could do:

```jsx
<Nav.Link as={NavLink} to={url}>Text</Nav.Link>
```